tests/integration.rs

```diff
@@ -2264,6 +2279,48 @@ mod tests {
             });
         }
 
+        #[cfg(target_arch = "x86_64")]
+        #[cfg_attr(not(feature = "mmio"), test)]
```

Comment:
Why is this excluded from mmio?

Reply:
Because it's using the FW.

Reply:
Does it need to use the FW? The FW has a much slower boot.

Reply:
ACPI + MMIO is not a supported configuration (last time I tried it there were issues.) For an integration test perspective we lump mmio and !acpi together so I think it's best to stick with what you've got.

Reply:
Alright!